scripts/snippets.py

```diff
@@ -238,6 +238,22 @@ def process_snippets(args: argparse.Namespace) -> Snippets:
 
     return snippets
 
+def process_snippets_inter(requested_snippets, snippet_roots) -> Snippets:
```

Comment:
I think this would be better named `find_snippets_in_roots` or something like that. I had a hard time guessing what `process_snippets_inter` meant when I first saw it earlier in the patch.